mlir/test/e2e/lit.cfg.py

```diff
@@ -33,6 +33,7 @@
 config.substitutions.append(("%mlir_src_root", config.mlir_src_root))
 config.substitutions.append(('%random_data', config.random_data))
 config.substitutions.append(('%xdlops', config.xdlops))
+config.substitutions.append(('%no_advanced_feature', config.no_advanced_feature))
```

Comment:
And, if we don't go with the above generality comment, perhaps name this something like `%force_features` or `%disable_features`?